Automattic/wp-calypso
# Connections: Change Instagram "Widget" to "Block"

Closes https://github.com/Automattic/wp-calypso/issues/66730

## Proposed Changes

This PR changes the mention of `Instagram Widget` to `Latest Instagram Posts` block.


## Testing Instructions

* Navigate to the Calypso Live Link: http://calypso.localhost:3000/marketing/connections/{yourSite}
* Scroll to the bottom of the page to the Instagram section
* Confirm that the text says: `Add the Latest Instagram Posts` block and the link goes to: https://wordpress.com/support/instagram/#embed-a-feed-of-instagram-posts

<img width="906" alt="Screenshot 2024-09-06 at 2 01 44 PM" src="https://github.com/user-attachments/assets/c0af5322-b9ac-4f98-b840-d724780187d3">

## Pre-merge Checklist
